pkg/configv2/migrate.go

```diff
@@ -0,0 +1,338 @@
+		// not been made https://www.notion.so/Rethinking-Configuration-435fbe87419148b4bbc5119d413786eb?d=1c73cb661b18405b8b67b3856f02e1d1&pvs=4#fdc6de7c00364de1bb81b325686e6e66
+		// It sounds like we only want to allow the setting of a default publisher, but unsure if this is on a job
+		// type basis or something else
+		JobDefaults: types.JobDefaults{
+			Batch: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+			Daemon: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+			Service: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
+				Task: types.TaskDefaultConfig{
+					Resources: types.Resource{
+						CPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.CPU,
+						Memory: in.Node.Compute.Capacity.DefaultJobResourceLimits.Memory,
+						Disk:   in.Node.Compute.Capacity.DefaultJobResourceLimits.Disk,
+						GPU:    in.Node.Compute.Capacity.DefaultJobResourceLimits.GPU,
+					},
+					Publisher: types.DefaultPublisherConfig{
+						Type: in.Node.Requester.DefaultPublisher,
+					},
+					Timeouts: types.TaskTimeoutConfig{
+						ExecutionTimeout: types.Duration(in.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
+					},
+				},
+			},
+			Ops: types.JobDefaultsConfig{
+				// TODO(forrest) [review]: Does a zero publisher value imply it has no priority or that it's not configured?
+				// Priority: 100,
```

Comment:
@wdbaruni what does a priority of `0` mean for scheduling? If we don't set these values `0` will be their default, so I think we need to arrive on a default value here. Or as my other comments have suggested, we can simplify this configuration to be for all jobs, and and job type configuration later.

Reply:
0 just means lowest priority. I think it should be the default instead of 50 to align with empty value defaults. I am okay with having defaults for all job types together instead of per job type for now